src/NuGetPackageVerifier/CompositeRules/DefaultCompositeRule.cs

```diff
@@ -23,6 +23,7 @@ public class DefaultCompositeRule : IPackageVerifierRule
             new SatellitePackageRule(),
             new StrictSemanticVersionValidationRule(),
             new DependenciesVersionRangeBoundsRule(),
+            new DotNetCliToolPackageRule(),
```

Comment:
I know we don't use any other rule list, but it would be nice to add new rules to other lists like `allRules`. It helps with debugging.